lib/ecies/aes128cbc.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+var bitcore = require('../../')
+var $ = bitcore.util.preconditions
+var aesjs = require('aes-js')
+var CBC = aesjs.ModeOfOperation.cbc
+
+var AES128CBC = function AES128CBC () {}
```

Comment:
what's the difference between this package and the other one called AESCBC? they look extremely similar, except that you are using the CBC package from aesjs instead of the built-in CBC package.